File: Programmers/Lv0/분수의덧셈.py
from fractions import Fraction
def solution(numer1, denom1, numer2, denom2):
    denom = denom1 * denom2 #통분
    numer1 *= denom2
    numer2 *= denom1
    numer = numer1 + numer2 #덧셈진행
    ret = Fraction(numer, denom)
    return [ret.numerator, ret.denominator]

File: Programmers/Lv0/분수의덧셈.cpp
#include <string>
#include <vector>
#include <numeric>
using namespace std;

vector<int> solution(int numer1, int denom1, int numer2, int denom2) {
    int denom = denom1 * denom2;
    numer1 *= denom2;
    numer2 *= denom1;
    int numer = numer1 + numer2;

    int _gcd = gcd(numer, denom);
    vector<int> answer;
    answer.push_back(numer/_gcd);
    answer.push_back(denom/_gcd);
    return answer;
}